src/Adapter/PDFLib.php

```diff
@@ -726,17 +726,14 @@ protected function _load_font($font, $encoding = null, $options = "")
         // Fix for PDFLibs case-sensitive font names
         $baseFont = basename($font);
         $isNativeFont = false;
-        if (isset(self::$nativeFontsTpPDFLib[$baseFont])) {
-            $font = self::$nativeFontsTpPDFLib[$baseFont];
+        $lcBaseFont = strtolower($basefont);
```

Comment:
Variable `$baseFont` is misspelled here.

Reply:
really need to get my environment fixed up (or my eyes)